react/transforms/address.ts

```diff
@@ -62,15 +62,15 @@ export function removeValidation(address) {
 export function addNewField<FieldName extends keyof ValidatedField>(
   address: AddressWithValidation,
   fieldName: FieldName,
-  value: ValidatedField[FieldName]
+  value: ValidatedField[FieldName] | ((fieldValue: ValidatedField) => boolean)
```

Comment:
oh, one comment I forgot to make was to substitute the `boolean` here by the `ValidatedField[FieldName]`, that makes sense?

```suggestion
  value: ValidatedField[FieldName] | ((fieldValue: ValidatedField) => ValidatedField[FieldName])
```